### PR TITLE
Refactor parallel package with passing excludedTests via Options.

### PR DESCRIPTION
**Pull request by VitalyGushin**
https://github.com/networkservicemesh/integration-tests/issues/3510

### CODE REVIEW

**Comment by denis-tingaikin** [comment]
@VitalyGushin Oops. Missed an update notification about this PR. Seems like linter job has failed. Could you please apply fixes for the linter?

**Comment by VitalyGushin** [comment]
@denis-tingaikin Linter predictably crashes when using the parallel.WithExcludedTests option, which has not yet been merged: https://github.com/networkservicemesh/integration-tests/pull/3513/files#diff-839274d618f2f2fe4dab902d0788f81b06cbbcd0e38248e9bffdb8fa7ba132aaR27

**Comment by denis-tingaikin** [comment]
@VitalyGushin Got it. Fixed.

**Review by denis-tingaikin — APPROVED** [review]

File: tests_ovs_extended/feature_ovs_test.go [hunk 1]
```diff
@@ -40,8 +42,16 @@ func (s *kindFeatOvsSuite) BeforeTest(suiteName, testName string) {
...
+			parallel.WithRunningTestsSynchronously(
+				featureOvsSuite.TestScale_from_zero,
+				featureOvsSuite.TestSelect_forwarder))
 	} else {
-		parallel.Run(t, new(features_ovs.Suite), "TestScale_from_zero", "TestNse_composition", "TestSelect_forwarder")
+		featureOvsSuite := new(features_ovs.Suite)
+		parallel.Run(t, featureOvsSuite,
+			parallel.WithRunningTestsSynchronously(
+				featureOvsSuite.TestScale_from_zero,
+				featureOvsSuite.TestSelect_forwarder))
```
Comment (denis-tingaikin) [inline tests_ovs_extended/feature_ovs_test.go:55]: Looks like TestNse_composition is missed

Comment (VitalyGushin) [reply tests_ovs_extended/feature_ovs_test.go:55]: it looks like he's gone

**Comment by denis-tingaikin** [comment]
The  changes are not related to the tests failing. So merging.

**Pull request merged**